com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -132,6 +132,11 @@ public enum FrameSettingsField
         /// <summary>When enabled, HDRP renders custom passes contained in CustomPassVolume components.</summary>
         [FrameSettingsField(0, autoName: CustomPass, customOrderInGroup: 11, tooltip: "When enabled, HDRP renders custom passes contained in CustomPassVolume components.")]
         CustomPass = 6,
+#if ENABLE_VIRTUALTEXTURES
```

Comment:
This framesettings must not but #ifdef, it should always be present (but greyd if VT is not enabled), otherwise serialization will fuck us.